modules/custom/moj_resources/src/SeriesContentApiClass.php

```diff
@@ -19,31 +20,42 @@ class SeriesContentApiClass
    *
    * @var array
    */
-  protected $nids = array();
+  protected $node_ids = array();
```

Comment:
🤩 loving the switch to more meaningful array names ❤️

Reply:
Definitely helps, there was a variable named `results` which turned out to be a `query`, something we should be mindful of as we go forward is fixing these variable names

Reply:
It was a good call by @LucasCairns to do that 😄 We need to do it across all the code.